### PR TITLE
AzureKiller: Clean Azure resources automatically

### PR DESCRIPTION
**Pull request by qwordy**
AzureKiller: Clean Azure resources automatically

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@haroldrandom  please help review the PR.

File: scripts/clean.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+
+def main():
+    print('Azure Killer, version 0.1')
+    print('Configuration:')
+    print('    Subscription: ' + subscription)
+    print('    Resource group prefix: ' + str(prefixes))
+    print('    Maximum survival time: %d days' % TTL)
+    print()
+    cmd = 'az group list --subscription %s --query [].name'
```
Comment (fengzhou-msft) [inline scripts/clean.py]: I ran this command locally. It gives me `zsh: no matches found: [].name`. Is it some known issue with zsh?

Comment (qwordy) [reply scripts/clean.py]: I only tested it in Powershell. It's JMESPath syntax

**Comment by qwordy** [comment]
> 
> 
> Since it got placed in azure-cli/scripts, I think the name "clean.py" is much ambiguous.
> Can we change to a more straightforward name?

No problem. You can modify it in this PR. Do you have any better name?

**Comment by haroldrandom** [comment]
> > Since it got placed in azure-cli/scripts, I think the name "clean.py" is much ambiguous.
> > Can we change to a more straightforward name?
> 
> No problem. You can modify it in this PR. Do you have any better name?

How about _azure_sdk_test_env_resoruce_cleaner.py_ ?

**Comment by qwordy** [comment]
> 
> 
> > > Since it got placed in azure-cli/scripts, I think the name "clean.py" is much ambiguous.
> > > Can we change to a more straightforward name?
> > 
> > 
> > No problem. You can modify it in this PR. Do you have any better name?
> 
> How about _azure_sdk_test_env_resoruce_cleaner.py_ ?

Awesome!

**Comment by jiasli** [comment]
We can put this in https://github.com/Azure/azure-cli-pr/wiki, as this is unrelated to Azure CLI.

**Pull request closed without merging**